lib/quby/answers/services/score_calculator.rb

```diff
@@ -230,6 +230,22 @@ def table_lookup(table_key, parameters)
           table.lookup(parameters)
         end
 
+        # Public: Ensure certain questions have an answer
```

Comment:
Ensure given question_keys have answers.
Strings with nothing but whitespace are not considered answered.